README.md

```diff
@@ -2044,7 +2044,8 @@ Parses the authorization code, access, or ID Tokens from the URL after a success
 
 If an authorization code is present, it will be exchanged for token(s) by posting to the `tokenUrl` endpoint.
 
-The ID token will be [verified and validated](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L186-L190) before available for use.
+The ID token will be [verified and validated](https://github.com/okta/okta-auth-js#tokenverifyidtokenobject) before available for use.
+In case access token is a part of OAuth2 flow response, its hash will be checked against ID token's `at_hash` claim.
```

Comment:
When would an access token not be part of an OAuth2 flow response?   Did you mean OIDC?  Or am I missing something?  (entirely possible)

Reply:
thanks, mentioning OAuth2 flow here doesn't seem to be right